### ISSUE TITLE
Broken dependencies on Debian 9 Stretch (libc6, libcurl4, libqt5core5a)

### ISSUE THREAD

**Issue by HannanAkbar** [TheAssassin/AppImageLauncher#216]
Tried both releases, continuous and release. It seems `bionic_amd64.deb` depends on 'Buster' packages. Where can I find a deb for Stretch?

```
The following packages have unmet dependencies:
 appimagelauncher : Depends: libc6 (>= 2.27) but 2.24-11+deb9u3 is installed
                    Depends: libcurl4 (>= 7.16.2) but it is not installable
                    Depends: libqt5core5a (>= 5.9.0~beta) but 5.7.1+dfsg-10deepin is installed
E: Unmet dependencies. Try 'apt --fix-broken install' with no packages (or specify a solution).
```

**Comment by TheAssassin** [comment]
Please try the `xenial` binaries and report back. If they work, I'll adjust the compat table in the README. I'm pretty sure I made a mistake there, `stretch` is older than `bionic` so I probably just mixed up two releases when putting together that list.

**Comment by TheAssassin** [comment]
So, it's quite strange that neither version can be installed:

![screenshot_2019-07-31_00-44-59](https://user-images.githubusercontent.com/4503202/62170665-86063f00-b32c-11e9-993a-88fa1402387a.png)
![screenshot_2019-07-31_00-44-37](https://user-images.githubusercontent.com/4503202/62170666-88689900-b32c-11e9-93bb-af93e3ee03f0.png)

Both are too similar to be a problem with Debian. I'm going to double-check that the `bionic` version is actually built on `bionic`, same for `xenial`.

**Comment by TheAssassin** [comment]
So, the released binaries are broken, and I know why (some builds were just done on `bionic` accidentally). That has been fixed already, and we're going to make a release soon-ish, and the new `xenial` binaries will then install just fine on `stretch`. I just tried that.

**Comment by TheAssassin** [comment]
The just released 1.4.0-beta-1 installs as expected. I also fixed the compat table. You should be able to install the `xenial` version now. Sorry for the inconveniences.
